pkg/controller/framework/informers/factory.go

```diff
@@ -42,3 +42,22 @@ func CreateSharedPodInformer(client clientset.Interface, resyncPeriod time.Durat
 
 	return sharedInformer
 }
+
+// CreateSharedIndexPodInformer returns a SharedIndexInformer that lists and watches all pods
+func CreateSharedIndexPodInformer(client clientset.Interface, resyncPeriod time.Duration) framework.SharedIndexInformer {
```

Comment:
`CreateSharedPodIndexInformer`?

Also, is the previous `CreateSharedPodInformer` used anywhere?  If not, I'd remove it.